### PR TITLE
importer: add link to found entities

### PR DESCRIPTION
**Pull request by jum-s**
solve #249

### CODE REVIEW

**Comment by maxlath** [comment]
* The problem with converting a candidate into an entity model (as done in `this.editionModel = new Entity(this.model.attributes)`) is that being an invalid or incomplete candidate is an accepted state (`isInvalid=true` or `needInfo=true`), while it's not for an entity, which seems to create different issues.
  * You can test an invalid candidate by passing an invalid ISBN http://localhost:3009/add/import?isbns=9780552167742
  * You can test an incomplete candidate by passing a valid but locally unknown ISBN http://localhost:3009/add/import?isbns=9782501151801 . I suspect that in this case, once info is entered by the user, you might need to handle the case where there is no entity associated with the candidate: the added info currently are only unstructured strings, that are later reconciled with the existing entities.
* Selected candidates have lost their green background, which was imo a helpful clue
* I would rather not make those links `target=_blank` as that breaks the general convention for internal links to open in the same tab: it should be fine to open in the same tab by default (can be done by just adding the class `showEntity` to the `a` elements), as long as hitting the previous button or coming back to the import page recovers the previous state. It's then still possible to open in a new tab by the standard means.
* I guess it's fine to not have the whole bar selectable, but a lot of users would probably enjoy having a larger click zone: having at least 1em around the checkbox clickable would be appreciable

**Comment by jum-s** [comment]
I think i disagree working on the last point on this branch, it would be appreciable indeed but its an issue in itself as the client could have a common way to select things through checkboxes (ie. shelves, and other examples i cant think of now)
Will fix all the other points

**Comment by jum-s** [comment]
tries to only convert a candidate into an entity model when necessary, but i could not find how to test it properly enough : 
- i could not incomplete candidate because dataseeds are powerful, 
- and passing an invalid candidate trigger the expected behavior => that is :

File: app/modules/inventory/views/add/candidate_row.js [hunk 1]
```diff
@@ -17,13 +18,21 @@ export default Marionette.ItemView.extend({
     return base
   },
 
+  initialize () {
+    if (this.isSelectable()) { this.editionModel = new Entity(this.model.attributes) }
```
Comment (jum-s) [inline app/modules/inventory/views/add/candidate_row.js:22]: is that better ? only actual selectable candidates are becoming entities

**Pull request closed without merging**